### ISSUE TITLE
add bias correction method validation to global validation 

### ISSUE THREAD

**Issue by dgergel** [ClimateImpactLab/downscaleCMIP6#381]
We also want to add some bias correction method validation to our global validation routine for a limited subset of gridcells. We need this to check two main things: 

1) that we preserve model-projected relative changes in quantiles 
2) that historical model bias corrected distributions match reference distributions 

See p. 6942 of Cannon et al 2015 and esp. eqtn (6) for additional reference/context on this

**Comment by emileten** [comment]
@dgergel thanks !

**Comment by dgergel** [comment]
@emileten can you link the method validation you did here? I think we can close this once you do

**Comment by emileten** [comment]
@dgergel yep. Eventually as you might have noticed we didn't do this systematically, but rather in a series of quick checks on notebooks shared on slack. 

I will look for the output I shared related to this specific check and upload it here.
